tests/pytests/integration/ssh/test_py_versions.py

```diff
@@ -135,6 +135,7 @@ def test_py36_target(salt_ssh_cli):
     Test that a python >3.6 master can salt ssh to a <3.6 target
     """
     ret = salt_ssh_cli.run("test.ping", minion_tgt="pyvertest")
+    if "kex_exchange_identification" in ret.stdout:
```

Comment:
Is this occurring each time on ubuntu 22.04?

Reply:
Not each time, it seems to happen randomly